introducing-java/src/main/java/com/onto/javacourse/intro/gcd/GCD.java

```diff
@@ -1,5 +1,42 @@
-public class Main {
-    public static void main(String[] args) {
+public class GCD {
+    /**
+     * Calculates the greatest common divisor of two numbers.
+     *
+     * @param firstNumber  The first integer.
+     * @param secondNumber The second integer.
+     * @return The greatest common divisor of the two numbers.
+     * @throws IllegalArgumentException If both numbers are 0 or only one of them is 0.
+     */
+    public static int calculateGcd(int firstNumber, int secondNumber) {
+        if (firstNumber == 0 || secondNumber == 0) {
+            throw new IllegalArgumentException("Invalid numbers, they cannot be 0.");
+        } else {
```

Comment:
My idea was to remove the else clause, you can do that when throwing an exception

```
        if (firstNumber == 0 || secondNumber == 0) {
            throw new IllegalArgumentException("Invalid numbers, they cannot be 0.");
        }
        while ....
```